### ISSUE TITLE
Adding modules to pytorch hub

### ISSUE THREAD

**Issue by bkkaggle** [pytorch/hub#45]
Hi,

I built a library called [pytorch_zoo](https://github.com/bkahn-github/pytorch_zoo) with a collection of pytorch modules, losses, schedulers, and utilities that I use often. I'd like to add some of these modules to pytorch hub.

Are you interested in contributions like this for pytorch hub, or is this repository only for pretrained models?

Thanks,

Bilal

**Comment by soumith** [comment]
which of the modules are you thinking to add to hub? I dont see any models from a glance at the README

**Comment by bkahn-github** [comment]
For modules, I have:

- [channel wise squeeze and excitation](https://github.com/bkahn-github/pytorch_zoo#squeezeandexcitationin_ch-r16)
- [Channel Squeeze and Spatial Excitation](https://github.com/bkahn-github/pytorch_zoo#channelsqueezeandspatialexcitationin_ch)
- [Concurrent Spatial and Channel Squeeze and Channel Excitation](https://github.com/bkahn-github/pytorch_zoo#concurrentspatialandchannelsqueezeandchannelexcitationin_ch)

For schedulers, I have:

- [Cyclical momentum](https://github.com/bkahn-github/pytorch_zoo#cyclicalmomentumoptimizer-base_momentum08-max_momentum09-step_size2000-modetriangular)

And for losses, I have:

- [lovasz-hinge loss](https://github.com/bkahn-github/pytorch_zoo#lovasz_hingelogits-labels-per_imagetrue)

I also a [dynamic batch length sampler](https://github.com/bkahn-github/pytorch_zoo#dynamicsamplersampler-batch_size32) which still needs some work to seamlessly work with pytorch

I was also interested in adding some of the small utilities, like notifications, and setting random seeds. 

These aren't pretrained pytorch models, so they might not be compatible out-of-the-box with hubconf.py files

**Comment by soumith** [comment]
@bkkaggle i think you missed the goal of Hub. It is for pre-trained pytorch models, that reproduce a paper, or showcase a functionality. In some cases these are also fine-tunable.

In your case, pytorch_zoo is more like a set of batteries.
